### PR TITLE
Feat/gym request form

### PR DESCRIPTION
**Pull request by yarkhinephyo**
#### Summary
- Usage of `<ion-grid>` in `/gyms` page
- Add `/gyms/request` page
- Update `requestGym` backend parameters and swagger

#### Flow
Go to `/gyms` page, then click on `Can't find gym`.
If you are logged in, you will be routed to gym request form.
If not, you are routed to log in page.

#### Note
Logging out gives error because of clearing of localStorage before `/logout` request. However, it is fixed in #84 

#### Diagram
image

### CODE REVIEW

File: lambda_backend/route_microservice/src/common/schema.ts [hunk 1]
```diff
@@ -120,7 +120,11 @@ export const addCommentSchema = {
       type: 'object',
       properties: {
         ...getRouteDetailsSchema.properties.body.properties,
-        comment: { type: 'string', maxLength: 150 },
+        comment: {
+          type: 'string',
+          maxLength: 150,
+          pattern: AsciiCharacters,
```
Comment (qwoprocks) [inline lambda_backend/route_microservice/src/common/schema.ts:126]: No emojis?

File: lambda_backend/route_microservice/src/common/schema.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ export const createRouteSchema = {
       properties: {
         countryCode: {
           type: 'string',
-          maxLength: 50,
+          maxLength: 3,
           pattern: AlphanumericSpaceHyphen,
```
Comment (qwoprocks) [inline lambda_backend/route_microservice/src/common/schema.ts:16]: If we're using ISO country code won't the pattern just be alpha/alphanumeric?

**Review by qwoprocks — APPROVED** [review]
Some comments

File: lambda_backend/route_microservice/src/common/schema.ts (hunk 1 above)
Comment (yarkhinephyo) [reply lambda_backend/route_microservice/src/common/schema.ts:126]: good catch

File: lambda_backend/route_microservice/src/common/schema.ts (hunk 2 above)
Comment (yarkhinephyo) [reply lambda_backend/route_microservice/src/common/schema.ts:16]: yea but i think its okay LOL

File: lambda_backend/route_microservice/src/common/schema.ts (hunk 1 above)
Comment (yarkhinephyo) [reply lambda_backend/route_microservice/src/common/schema.ts:126]: Will make a new issue

File: lambda_backend/route_microservice/src/common/schema.ts (hunk 2 above)
Comment (yarkhinephyo) [reply lambda_backend/route_microservice/src/common/schema.ts:16]: fixed

**Pull request merged**